packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue

```diff
@@ -106,4 +106,10 @@ export default defineComponent({
   box-sizing: border-box;
   color: currentColor;
 }
+
+.va-sidebar:not(.va-sidebar--animated) {
+  .va-sidebar__item {
+    transition: none;
```

Comment:
This will turn off hover animation. Feature ?

Reply:
Removed this rule